packages/docs/pages/button.tsx

```diff
@@ -1,145 +1,221 @@
-              components will be removed.
-            </Text>
-
-            <CodePreview>
-              {/* jsx-to-string:start */}
-              <>
-                <Button iconOnly={<AddIcon title="add" />} />
-                <Button iconLeft={<AddIcon />}>Label</Button>
-                <Button iconLeft={<AddIcon />} iconRight={<ArrowDropDownIcon />}>
-                  Label
-                </Button>
-                <Button iconRight={<ArrowDropDownIcon />}>Label</Button>
-              </>
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-        </>
-      ),
-    },
-    {
-      id: 'props',
-      title: 'Props',
-      render: () => <ButtonPropTable inheritedProps={<MarginPropTable collapsible />} />,
-    },
-  ];
-
   return (
     <>
       <H1>Button</H1>
 
-      <PageNavigation items={items} />
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          <Code primary>Buttons</Code> trigger an immediate action, which can continue within the current page, a new
+          page or modal.
+        </Text>
+        <Text bold>When to use:</Text>
+        <List>
+          <List.Item>
+            Use <Code primary>Primary Buttons</Code> for the most important action on a page (e.g. creating an object,
+            saving changes).
+          </List.Item>
+          <List.Item>
+            Use <Code primary>Secondary Buttons</Code> for other important page actions that are not the primary task on
+            the page.
+          </List.Item>
+          <List.Item>
+            Use <Code primary>Subtle Buttons</Code> alongside <Code primary>Secondary</Code> or{' '}
+            <Code primary> Primary Buttons</Code> for teritiary actions, like to cancel or visit a related task.
+          </List.Item>
+          <List.Item>
+            Use <Code primary>Destructive Buttons</Code> in confirmations for important actions that cannot be undone
+            (like deleting something).
+          </List.Item>
+        </List>
+      </Panel>
+
+      <Panel header="Implementation" headerId="implementation">
+        <ContentRoutingTabs
+          id="implementation"
+          routes={[
+            {
+              id: 'basic',
+              title: 'Basic',
+              render: () => (
+                <>
+                  <Text>Buttons are calls to action used throughout the product experience.</Text>
+                  <CodePreview>
+                    {/* jsx-to-string:start */}
+                    <Button actionType="normal" isLoading={false} variant="primary">
+                      Label
+                    </Button>
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </>
+              ),
+            },
+            {
+              id: 'variants',
+              title: 'Variants',
+              render: () => (
+                <Fragment key="variants">
```

Comment:
You can actually just leave these as fragments without keys `<></>`

Reply:
I tried to use the `<></>` but when using the fragments like that the `LiveEditor` wasn't working as expected. It was rendering the same `code` when switching between tabs.

Reply:
If that is the case we need to fix these in a bunch of pages, or maybe figure out why `LiveEditor` doesn't like that,

Reply:
sounds good, I'll check which pages are having the same issue

Reply:
Ah yeah good catch. I now remember having to do this in the `Form` documentation.